### DISCUSSION TITLE
Sight error inside an example

### DISCUSSION THREAD

**Discussion by garrosh** [robbievanleeuwen/section-properties#506] [Q&A] [answered]
So I was reading through this page : [Pilkey - Circular Arc](https://sectionproperties.readthedocs.io/en/latest/examples/validation/pilkey_arc.html) and I noticed a mistake in the text at the end. You say the biggest error is on J, but it is in fact on zxx because, unless I'm mistaken, you need to consider the absolute value of the error and not just the strictly positive one.

Thank you very much for your help!

**Comment by robbievanleeuwen** [comment, marked as answer]
Hi @garrosh, thanks for the comment.

The example reads:

> Out of the *warping properties*, the torsion constant had the largest relative error, however this value is relatively small and acceptable given the differences in element type and mesh.

While `zxx` does have the largest relative error, it is not a warping property, but an area property - so the above sentence is still true.
